examples/hogwart_objection/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "test-lib",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
These embedded example apps are great. I think for added clarity they could use `npm start` scripts and some very simple `README.md`s like:

``` markdown
Run this Example:
  1. `cd examples/hogwart_objection`
  2. `npm i`
  3. `npm start`
  
```

... and if we want to get crazy, maybe some very basic example requests so its clear what the example is demonstrating:
```markdown
Get all students:
`curl localhost:3000/students`

Get all students sorted by name ascending:
`...`
```

Reply:
Yeah, this is nice. I plan to write a README for all the extra ORM packages and examples. The examples will not be usable yet till we publish the other ORMS as actual packages. This is noted!